### ISSUE TITLE
Allow more flexibility with branch names created from issues

### ISSUE THREAD

**Issue by dan1994** [KnisterPeter/vscode-github#656]
I personally like to name my branches using the issue number and the issue name.
For example if my issue 34th issue is "implement tests for module x", I would name the corresponding branch `34-implement-tests-for-module-x`.

The current `githubIssues.workingIssueBranch` setting only lets you use `user` and `issueNumber` as placeholder, which is not enough to create my naming scheme.

Of course that mine is only an example. Some also like to put in info like if that's a feature, bugfix, etc., which you can extract for labels or other issue fields.

In short, my feature request would be to allow more placeholders for the `githubIssues.workingIssueBranch` setting.

**Comment by KnisterPeter** [comment]
I think you are repoting to the wrong extension. This one does not have a `githubIssues` setting.

**Comment by dan1994** [comment]
Sorry about that...
